### ISSUE TITLE
Need help to create  virtual  Hook trade counter .

### ISSUE THREAD

**Issue by BINARYbot00** [binary-com/binary-bot#3644]
How i can define custome virtual Hook counting . 
 
Means 
In virtual Hook 
If
 win loss loss = count  1
Win loss loss = count 2 
Win loss =  trade on real account 
Do you have any knowledge about it 
@binary-ex-machina
Give me image of text file i need guidance to create it.

**Comment by derivmaster** [comment]
My friend @BINARYbot00 this was a difficult job but is READY  !!! 💪

![image](https://user-images.githubusercontent.com/107600731/173999525-543ad04f-149d-4a9d-9a5e-10bae82c6f23.png)

YOU ONLY NEED TO ADD THE STRATEGY YOU WANT AND WILL BE READY ! 

![image](https://user-images.githubusercontent.com/107600731/174002880-97c20ee0-fb7c-4334-bb2a-afec28fffc50.png)


The Bot ( change the .txt for .xml ) -

[Virtual Hook Patterns 2022 (Binatytools).txt](https://github.com/binary-com/binary-bot/files/8916132/Virtual.Hook.Patterns.2022.Binatytools.txt)

Remember RUN the bot here 👉🏻 **BOT.BINARYTOOLS.IO**

![image](https://user-images.githubusercontent.com/107600731/173999640-644ae03c-3796-41ba-8a0c-006e974b33d6.png)

**Comment by BINARYbot00** [comment]
Thanks for giving your time to develope our idea to binary bot  if there is
something that we can do it for you please don't hesitate to contact us.
Good wishes from us bless you .
Regards,
Binary tricks
